docs/docs/crdt-details.md

```diff
@@ -12,7 +12,7 @@ This document contains a detailed description of the data structures and operati
 
 Beyond synchronizing text, these data structures and operations allow Xi to handle asynchronous editing of the text from plugins, support undo and redo, and allow incremental updating of editor state and the view based on differences between revisions.
 
-Many of these data structures and operations have been in Xi for a while but they've recently been heavily overhauled and extended as part of [a project](https://github.com/google/xi-editor/issues/250) that added multi-device syncing support via the CRDT merge operation. This was done for [use on the Fuchsia operating system](https://fuchsia.googlesource.com/xi/), where it uses [Ledger](https://fuchsia.googlesource.com/ledger/) to synchronize documents between devices.
+Many of these data structures and operations have been in Xi for a while but they've recently been heavily overhauled and extended as part of [a project](https://github.com/google/xi-editor/issues/250) that added multi-device syncing support via the CRDT merge operation. This was done for [use on the Fuchsia operating system](https://fuchsia.googlesource.com/third_party/xi-editor/), where it uses [Ledger](https://fuchsia.googlesource.com/peridot/+/HEAD/docs/ledger) to synchronize documents between devices.
```

Comment:
I think the most appropriate link for Fuchsia xi is `https://fuchsia.googlesource.com/topaz/+/master/bin/xi/` ; the third party one is basically just a mirror of upstream xi and does not contain the front-end.

Reply:
Done.